### PR TITLE
Make contactable_by_* fields nullable

### PR DESCRIPTION
**Pull request by elcct**
### Description of change

This change makes `contactable_by_*` fields nullable as another step in removal.

### Checklist

* [X] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: changelog/contact/contact.db [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The column ```contact.contactable_by_dit``` is nullable.
```
Comment (reupen) [inline changelog/contact/contact.db]: I would reword slightly, to something like:

> The column ```contact.contactable_by_dit``` was made nullable in preparation for its removal.

**Comment by elcct** [comment]
I am going to close this one and prepare another with full removal. Thanks!
